doc/changelogs/CHANGELOG_V10.md

```diff
@@ -98,6 +102,7 @@
 * Util
   * `util.types.is[…]` type checks have been added. [[`b20af8088a`](https://github.com/nodejs/node/commit/b20af8088a)]
   * Support for bigint formatting has been added to `util.inspect()`. [[`39dc947409`](https://github.com/nodejs/node/commit/39dc947409)]
+  * Runtime deprecate using a property named `inspect` on an object to specify a custom inspection with `util.inspect()`. [[`617e3e96e6`](https://github.com/nodejs/node/commit/617e3e96e6)]
```

Comment:
``  Using a property... has been deprecated.`` or `` Runtime deprecation of using... ``?

Reply:
Hm, I tried to reword it as suggested but it still sounds a bit off to me.

Reply:
`util.inspect()` custom inspection with `inspect` property has been deprecated at runtime.

?